test/Altinn.Notifications.Tests/Notifications/TestingMappers/OrderMapperTests.cs

```diff
@@ -72,7 +72,7 @@ public void MapToNotificationOrderExt_AreEquivalent()
     }
 
     [Fact]
-    public void MapToRecipientExt_AllPropertiesPresent_AreEquivalent()
+    public void MapToRecipientExt_ForEmailAllPropertiesPresent_AreEquivalent()
```

Comment:
Could we keep the original name and add the sms adress point to the address list instead of duplicating the test?

Reply:
Well, Keeping separate test makes it easier to point out where the failure is happening in the case of failure. Of course we have error message, but when the codebase is getting bigger and complicated it gets hard to debug with error message and in that case separate tests comes quite handy. 🤔 

But I'm also curious about what we can achieve by not duplicating the test, or making one test instead of two? 🤔

Reply:
We will at some point support sending through both communication channels. So we also need to confirm that the combination of email and sms works. If the combination works we can trust that they will work separately as well. So I would say 1 test is sufficient if not we should have 3 testing email, sms and both.